app/src/main/java/com/kms/cura/constant/DataConstant.java

```diff
@@ -0,0 +1,79 @@
+package com.kms.cura.constant;
+
+import com.kms.cura.entity.DegreeEntity;
+import com.kms.cura.entity.FacilityEntity;
+import com.kms.cura.entity.SpecialityEntity;
+import com.kms.cura.event.EventHandler;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Created by linhtnvo on 6/16/2016.
+ */
+public class DataConstant {
```

Comment:
Not really "Constant" here. This one manage all entities that should not changed a lot, then it should be controller and put in controller package.